### ISSUE TITLE
Error while exporting notebooks

### ISSUE THREAD

**Issue by YatharthKaushik** [databrickslabs/migrate#154]
Hey,
I am facing the following error while exporting notebooks and running the following command: 
`python export_db.py --profile oldWS --workspace --azure`

<img width="666" alt="image" src="https://user-images.githubusercontent.com/49161779/161538234-6d6def45-46ee-4711-8359-2ac6fe3735b8.png">

I have cloned the most recent version of the repository.
Any help is appreciated.

Thanks!

**Comment by mrchristine** [comment]
The latest version of the master branch works for me. Can you run a `git status` and see if you have the right version?

**Comment by YatharthKaushik** [comment]
Hey, 
I was on the latest version of the master branch.
But I upgraded all the python packages using:
`pip list --outdated --format=freeze | grep -v '^\-e' | cut -d = -f 1  | xargs -n1 pip install -U`
And this resolved the error.

Thanks!

**Comment by dvatterott** [comment]
I'm hitting the same error. I think this is due to a version dependency for the requests library. @YatharthKaushik or @mrchristine could you post what version of requests you're using?

**Comment by dvatterott** [comment]
Seems related to #142 and #146 specifically - https://github.com/databrickslabs/migrate/pull/142/commits/c9fe8debb2e52a3d1ea44ebdb96b267190b0be04

**Comment by dvatterott** [comment]
Okay, I was able to chase this down. For anyone who hits this issue in the future. This is a urllib3 version dependency not requests (requests has some weird importing going on). I was on urllib 1.25.11 (https://urllib3.readthedocs.io/en/1.25.11/reference/urllib3.util.html#urllib3.util.retry.Retry) where Retry does not take the named argument "allowed_methods". The latest version of urllib3 has a version of Retry that takes "allowed_methods"(1.26.9; https://urllib3.readthedocs.io/en/1.26.9/reference/urllib3.util.html?highlight=retry#urllib3.util.Retry). 

TLDR - update urllib3.
